### PR TITLE
Problem: Missing mainnet address generation guide

### LINKED ISSUE #113: Problem: Missing mainnet address generation guide

### PR DESCRIPTION
**Pull request by calvinlauyh**
Fix #113 
Add mainnet address genereation doc

### CODE REVIEW

File: docs/wallets/mainnet-address-generation.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+Supported OS: Linux, Mac OS and Windows
+
+#### Step 1. Get the Crypto.org Chain binary
+
+Download the Crypto.org Chain Binary from Crossfire from [release page](https://github.com/crypto-org-chain/chain-main/releases/tag/v0.9.1-crossfire) and extract the binary. Here we used Linux as an example:
+
+```bash
+$ curl -LOJ https://github.com/crypto-org-chain/chain-main/releases/download/v0.9.1-crossfire/chain-main_0.9.1-crossfire_Linux_x86_64.tar.gz
+$ tar -zxvf chain-main_0.9.1-crossfire_Linux_x86_64.tar.gz
```
Comment (tomtau) [inline docs/wallets/mainnet-address-generation.md:46]: use 1.0.0 -- or will this be blocked by the log level issue mentioned on Slack?

Comment (calvinaco) [reply docs/wallets/mainnet-address-generation.md:46]: It was previously on hold because of it. But now it's resolved I will update to 1.0.0, thanks!

Comment (calvinaco) [reply docs/wallets/mainnet-address-generation.md]: Updated

File: docs/wallets/mainnet-address-generation.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+```
+
+-----
+
+## B-ii. Ledger Wallet via Crypto.org Chain Desktop Wallet
+
+::: warning DESKTOP WALLET IS IN BETA
+Desktop wallet is in beta testing. Please be aware of the [potential risks](https://github.com/crypto-com/chain-desktop-wallet#warning) of using it in mainnet. You should run it only on a trusted, safe and offline computer and always verify the mnemonic words before using the address.
+:::
+
```
Comment (lezzokafka) [inline docs/wallets/mainnet-address-generation.md:150]:
```suggestion
Download the latest version of the Crypto.org Chain Desktop Wallet [here](https://github.com/crypto-com/chain-desktop-wallet/releases) and follow the steps below to create an address:
```

File: docs/wallets/mainnet-address-generation.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+#### Step 2. After setting up the password, click "Create Wallet".
+
+<img src="./assets/mainnet-address-generation/desktop-create-wallet.png" />
+
+#### Step 3. Fill in any wallet name of your choice and choose "MAINNET" as the network. Then check "Want to create with hardware wallet?" and choose "Ledger" as "Wallet Type"
+
+<img src="./assets/mainnet-address-generation/ledger-desktop-create-wallet.png" />
+<img src="./assets/mainnet-address-generation/desktop-create-success.png" />
+
+#### Step 5. Upon successful wallet creation, a warning message will be shown on top right hand corner to remind you that the wallet is for address generation only and may not work when mainnet launch.
```
Comment (lezzokafka) [inline docs/wallets/mainnet-address-generation.md:164]:
```suggestion
#### Step 5. Upon successful wallet creation, a warning message will be shown on the top right hand corner to remind you that the wallet is for address generation only and may not work when mainnet launch.
```

File: docs/wallets/mainnet-address-generation.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+#### Step 4. Fill in any wallet name of your choice and choose "MAINNET" as the network. Then create the wallet by clicking "Create Wallet"
+
+<img src="./assets/mainnet-address-generation/desktop-create-mainnet-wallet.png" />
+<img src="./assets/mainnet-address-generation/desktop-create-success.png" />
+
+#### Step 5. Upon successful wallet creation, a list of 24 mnemonic words will be displayed. Make sure you have copied and stored your mnemonic words in a safe place.
+
+<img src="./assets/mainnet-address-generation/desktop-backup-mnemonic.png" />
+
+#### Step 5. A warning message will be shown on top right hand corner to remind you that the wallet is for address generation only and may not work when mainnet launch.
```
Comment (lezzokafka) [inline docs/wallets/mainnet-address-generation.md:236]:
```suggestion
#### Step 5. A warning message will be shown on the top right hand corner to remind you that the wallet is for address generation only and may not work when mainnet launch.
```

**Comment by lezzokafka** [comment]
I remember that one would need to shut down the Ledger Live app before using the desktop wallet , It that still the case @leejw51crypto ?

File: docs/wallets/mainnet-address-generation.md [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+```
+
+If you are downloading the binary for other operating systems, make sure you are downloading `v1.0.0`, which is the version targeting for mainnet.
+
+Before moving to the next step, kindly check your `chain-maind` version by 
+
+```bash
+$ ./chain-maind version
+1.0.0
+#### Step 2. Create a new key and address
```
Comment (lezzokafka) [inline docs/wallets/mainnet-address-generation.md:60]: @calvinaco Sorry I found it skipped the   "```"  bash-script-closing in my last suggestion inserted.

**Pull request merged**